web/src/components/avatar/group-item/styles.ts

```diff
@@ -33,6 +33,7 @@ export default createUseStyles(
       color: theme.colorTextAvatarItemSecondary,
```

Comment:
members class name already has a color for any text inside members block. We either need to remove color property from members class name, either do not add anything to membersCount class name